tests/nnapi/nnapi_gtest.skip.noarch.interp

```diff
@@ -23,6 +23,12 @@ GeneratedTests.argmax_neg_axis_float_nnfw
 GeneratedTests.argmax_neg_axis_int32_nnfw
 GeneratedTests.argmax_quant8_neg_axis_nnfw
 GeneratedTests.argmax_quant8_nnfw
+GeneratedTests.argmin_1
+GeneratedTests.argmin_1_quant8
+GeneratedTests.argmin_2
+GeneratedTests.argmin_2_quant8
+GeneratedTests.argmin_3
+GeneratedTests.argmin_3_quant8
```

Comment:
@hseok-oh Maybe beyond this PR. I wonder what happens if scale is negative. I cannot find any document or code that ensures `scale` will be positive. Then, `argmin` or `argmax` can bring the opposite result where `scale < 0`.

Reply:
Generally, scale is positive value.

- `NNAPI` 
https://github.com/Samsung/ONE/blob/948150008659635f5a8b969f14b31157ae9ce47c/runtime/nnapi-header/include/NeuralNetworks.h#L90-L93

- `TensorFlow Lite`: https://www.tensorflow.org/lite/performance/quantization_spec#symmetric_vs_asymmetric
referece paper: https://arxiv.org/pdf/1712.05877.pdf
page 2 `The constantS(for “scale”) is an arbitrary positive realnumber.`